src/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -750,8 +748,6 @@ public partial class XmlQualifiedName
         public override string ToString() { throw null; }
         public static string ToString(string name, string ns) { throw null; }
     }
-    [System.Diagnostics.DebuggerDisplayAttribute("{debuggerDisplayProxy}")]
-    [System.Diagnostics.DebuggerDisplayAttribute("{debuggerDisplayProxy}")]
```

Comment:
Excluding these attributes according to https://github.com/dotnet/corefx/blob/26b932e3c56702382d13ece010b300d3181d2891/eng/DefaultGenApiDocIds.txt#L13

Reply:
Keeping for now. https://github.com/dotnet/corefx/issues/35601